scene/3d/soft_body.cpp

```diff
@@ -700,6 +702,8 @@ SoftBody::SoftBody() :
 		simulation_started(false),
 		pinned_points_cache_dirty(true) {
 
+	ray_pickable = true;
```

Comment:
This should be initialized like other fields in the list.

Reply:
I "copied" it from here:
https://github.com/godotengine/godot/blob/99cee9038bf4fe049991d0048de8b8d8955bd9c7/scene/3d/collision_object.cpp#L342-L350